bin/doc/json.js

```diff
@@ -246,13 +246,11 @@ const processList = (section) => {
   // depending on the section type, the list could be different things.
 
   switch (section.type) {
-    case 'ctor':
-    case 'classMethod':
```

Comment:
Why were these two lines removed? Their removal changes the behavior.

Reply:
My mistake, thanks for spotting!